### PR TITLE
Fix: Remove import for undefined class

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] removes an import for an undefined class and fixes a related docblock

### CODE REVIEW

File: src/Framework/Constraint/Constraint.php [hunk 1]
```diff
@@ -92,9 +91,9 @@ public function count()
     /**
      * Throws an exception for the given compared value and test description
      *
-     * @param mixed                                          $other             Evaluated value or object.
-     * @param string                                         $description       Additional information about the test
-     * @param SebastianBergmann\Comparator\ComparisonFailure $comparisonFailure
+     * @param mixed             $other             Evaluated value or object.
+     * @param string            $description       Additional information about the test
+     * @param ComparisonFailure $comparisonFailure
```
Comment (localheinz) [inline src/Framework/Constraint/Constraint.php]: Class is already imported here, see [`Constraint.php#L17`](https://github.com/sebastianbergmann/phpunit/blob/b2e74111300b09f5a23e6213e8abbeac6536bb85/src/Framework/Constraint/Constraint.php#L17).

File: src/Framework/Constraint/Constraint.php [hunk 2]
```diff
@@ -12,7 +12,6 @@
 
 use Countable;
 use PHPUnit\Framework\ExpectationFailedException;
-use PHPUnit\Framework\SebastianBergmann;
```
Comment (localheinz) [inline src/Framework/Constraint/Constraint.php]: Class doesn't exist.

**Comment by sebastianbergmann** [comment]
Merged manually into `6.0` and `master`.

**Comment by localheinz** [comment]
Thank you, @sebastianbergmann!

**Pull request closed without merging**